doc/stand-alone.md

```diff
@@ -164,4 +164,16 @@ plugins:
   - name: jwt-auth
   - name: mqtt-proxy
     stream: true # set 'stream' to true for stream plugins
+#END
+```
+
+#### How to enable SSL
+
+```yml
+ssl:
+    -
+        cert: "...",
```

Comment:
We should use `literal_block` for cert and keys.
And it would be better to put full cert/key in the example.

Reply:
fixed.